eng/Tools.props

```diff
@@ -44,6 +44,7 @@
     <PackageReference Include="System.Runtime.Analyzers" Version="1.1.0" />
     <PackageReference Include="System.Runtime.InteropServices.Analyzers" Version="1.1.0" />
     <PackageReference Include="System.Security.Cryptography.Hashing.Algorithms.Analyzers" Version="1.1.0" />
+    <PackageReference Include="xunit.analyzers" Version="0.10.0" />
```

Comment:
We already packagereference xunit.analyzers here https://github.com/dotnet/corefx/blob/master/external/test-runtime/XUnit.Runtime.depproj#L34 (meta-package --> dependent package)

Reply:
We need to reference the analyzer's package directly in Tools.props as we generate a static props file that lists the Analyzer item generated by nuget automatically when running restore on Tools.proj -- I know it is already referenced, but we need it here because of the build flow and the way the analyzers are designed to be plumbed into corefx:

https://github.com/dotnet/corefx/blob/a4bd4039450c8017fc09582e514b64f02c2ae056/eng/Tools.props#L123